jf_agent/git/bitbucket_cloud_client.py

```diff
@@ -151,9 +145,7 @@ def get_all_pages(self, url, rate_limit_realm=None, ignore404=False):
                     page = self.get_json(url, rate_limit_realm)
                 except requests.exceptions.HTTPError as e:
                     if e.response.status_code == 404 and ignore404:
-                        agent_logging.log_and_print(
-                            logger, logging.INFO, f'Caught a 404 for {url} - ignoring',
-                        )
+                        logger.log(f'Caught a 404 for {url} - ignoring',)
```

Comment:
B: This will error, did you mean `logging.info`?

Reply:
Yup, that should be logger.info